src/pages/_app.page.tsx

```diff
@@ -8,6 +8,7 @@ import '@fortawesome/fontawesome-svg-core/styles.css';
 config.autoAddCss = false; // Tell Font Awesome to skip adding the CSS automatically since it's being imported above
 import { useRouter } from 'next/router';
 import LooMap from '../components/LooMap/LooMapLoader';
+import '../design-system/components/Badge/Badge.css';
```

Comment:
I think this style sheet will need to be included in the Badge story in Storybook for it to be styled correctly there (I've added comments to the two affected stories in the Chromatic visual diff tool as well)

I think something like this near the top in `.storybook/preview.js` will do the trick nicely:

```typescript
// Import design system component styles for our Stories.
import '../src/design-system/components/Badge/Badge.css';
```

We'll just need to make sure to keep both `_app.page.tsx` and `.storybook/preview.js` up to date with the latest component styles until we combine them all into a single design system stylesheet

Reply:
👍🏻 good spot, I hadn't checked Storybook after I fixed the css conflict error I was having. 

I've managed to import the css into the actual story (`./Badge.stories.tsx`), would there be an advantage to putting the css there rather than in `.storybook/preview.js`,  in that folk might be more likely to remember to add it, or is that an odd place to have it?

Reply:
If it's only imported in `./Badge.stories.tsx` we risk it working for that story, but we have a story which uses the Badge component such as the toilet details panel, the styles might not be available because they're coupled tightly to the Badge story only, not the ToiletDetailsPanel story.

Having it in `.storybook/preview.js` ensures that the badge styles are available to every story that we define (kinda analogous to putting into `_app.page.tsx` in the context of the Next app)

Reply:
👍🏻 That makes total sense, thank Ollie 😄